### PR TITLE
add disabled support for iOS

### PR DESCRIPTION
**Pull request by chrisdrackett**
# Summary

Adds ability to disable this control on iOS.

ref #303

## Test Plan

Added ability to toggle to example app

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅      |
| Android |    ❌     |

## Checklist

- [x] I have tested this on a device and a simulator
- [x] I added the documentation in `README.md`
- [x] I updated the typed files (TS and Flow)
- [x] I added a sample use of the API in the example project (`example/App.js`)
- [x] I have added automated tests, either in JS or e2e tests, as applicable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -360,6 +361,10 @@ This means that eg. if the device has dark mode turned on, and your screen backg
 <RNDateTimePicker style={{flex: 1}} />
 ```
 
+#### `disabled` (`optional`, `iOS only`)
+
+If true the user won't be able to toggle the switch.
```
Comment (vonovak) [inline README.md:366]:
```suggestion
If true, the user won't be able to interact with the view.
```

**Review by vonovak — CHANGES_REQUESTED** [review]
@chrisdrackett please just accept the suggested change and we're good to go, thank you!

**Comment by chrisdrackett** [comment]
@vonovak done!

**Review by vonovak — APPROVED** [review]

**Pull request merged**